src/python/WMCore/MicroService/MSTransferor/DataStructs/GrowingWorkflow.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Workflow object representing how growing workflows have to be dealt
+with in the input data placement (MSTransferor)
+"""
+
+from WMCore.MicroService.MSTransferor.DataStructs.Workflow import Workflow
+
+
+class GrowingWorkflow(Workflow):
+    """
+    Class to represent a Growing workflow in the context
+    of input data placement in MSTransferor
+    """
+
+    def getInputData(self):
+        """
+        Returns the primary dataset name to be locked and
+        transferred with Rucio, instead of a list of blocks.
+        :return: a list of unique blocks
+        """
+        inputContainer = [self.getInputDataset()]
+        currentTotalSize = sum([blockInfo['blockSize'] for blockInfo in self.getPrimaryBlocks().values()])
```

Comment:
I wanted to distinguish it from `totalBlockSize` because it actually is the total size at this very moment. As this input dataset will keep growing (new blocks will appear), I wanted to make it more explicit with the variable name.
But `totalBlockSize` isn't a bad idea for consistency.